client/templates/reporting/standard-reports/pool-search.njk

```diff
@@ -0,0 +1,203 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "custom-components/autocomplete/macro.njk" import modAutocomplete %}
+{% from "custom-components/date-picker/macro.njk" import datePicker %}
+
+{% block page_title %}
+  Reports
+{% endblock %}
+
+{% block page_identifier %}Reports - {{ title }}{% endblock %}
+
+{% set currentApp = "Reports" %}
+
+{% block content %}
+  {% include "includes/errors.njk" %}
+
+  <form method="post" id="poolSearchForm">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+
+        {% set poolNumberError = undefined %}
+        {% if errors.items["poolNumber"] %}
+          {% set poolNumberError = { html: errors.items["poolNumber"][0].details } %}
+        {% endif %}
+
+          {% call govukFieldset({
+            legend: {
+              text: title,
+              classes: "govuk-fieldset__legend--l",
+              isPageHeading: true
+            }
+          }) %}
+            <div class="govuk-grid-column-half">
+
+              {{ govukInput({
+                label: {
+                  text: 'Enter a pool number'
+                },
+                classes: 'govuk-!-width-one-third',
+                id: "poolNumber",
+                name: "poolNumber",
+                type: "text",
+                errorMessage: poolNumberError,
+                value: filter
+              }) }}
+
+            </div>
+            {# layout here is not quite right #}
+            <div class="govuk-grid-column-half govuk-button-group">
+
+              {{ govukButton({
+                text: "Search",
+                type: "submit",
+                attributes: {
+                  id: "searchButton",
+                  formAction: filterUrl
+                }
+              }) }}
+
+              <a class="govuk-link" href="{{ url(reportKey + '.filter.get') }}">Clear search</a>
+
+            </div>
+          {% endcall %}
+
+
+          <input type="hidden" name="_csrf" value="{{ csrftoken }}">
+          <input type="hidden" name="filter" value="{{ filter }}">
+      </div>
+    </div>
+
+    {% if resultsCount === 0 and errors.count === 0 and filter !== undefined %}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+          <hr class="govuk-section-break govuk-section-break--visible">
+          <h2 class="govuk-heading-m govuk-!-margin-top-4">There are no matching results</h2>
+          <p class="govuk-body">Improve your search by:</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>double-checking your spelling</li>
+          </ul>
+        </div>
+      </div>
+    {% elif resultsCount > 0 %}
+
+      {% set poolRows = [] %}
+      {% for pool in poolList %}
+        {% set poolRows = (poolRows.push(
+          [
+            {
+              html: '<div class="govuk-radios__item govuk-!-padding-left-0">' +
```

Comment:
what if you make these `govuk-radios--small`? 😄  I think it would align better

Reply:
This was not the trivial fix it feels like, so I have not. :)